rest/handler.go

```diff
@@ -440,11 +442,7 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 					// _default doesn't exist for a non-default scope - so make it a required element if it's ambiguous
 					return base.HTTPErrorf(http.StatusBadRequest, "Ambiguous keyspace: %s.%s", base.MD(keyspaceDb), base.MD(base.StringDefault(keyspaceScope, "")))
 				}
-				// one collection only - use that one
-				for collectionName := range scope.Collections {
-					keyspaceCollection = &collectionName
-					break
-				}
+				keyspaceCollection = base.StringPtr(base.DefaultCollection)
```

Comment:
There isn't a default collection on a non-default scope, and I don't see where we're setting keyspaceScope to base.DefaultScope in this path.

Reply:
I'm going to catch this in https://issues.couchbase.com/browse/CBG-2654